### PR TITLE
chore(toggle): state model

### PR DESCRIPTION
**Pull request by lewisbirks**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/projects/FPLA/versions/35300

### Change description ###

Toggling on tickets from the above release

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/CaseTypeTab/CareSupervision/4_DraftOrders.json [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "DraftOrdersTab",
+    "TabLabel": "Draft orders",
+    "TabDisplayOrder": 4,
+    "CaseFieldID": "draftSDOLinkDraftOrdersTab",
+    "TabFieldDisplayOrder": 3,
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseTypeTab/CareSupervision/4_DraftOrders.json:32]: are the numbers correct, they are repeated in cmo part

Comment (tomaszpowroznik) [reply ccd-definition/CaseTypeTab/CareSupervision/4_DraftOrders.json:32]: but, yeah, it was signed off this way

Comment (lewisbirks) [reply ccd-definition/CaseTypeTab/CareSupervision/4_DraftOrders.json:32]: CMO and SDO won't be there at the same time so it is okay for the numbers to be the same

Comment (tomaszpowroznik) [reply ccd-definition/CaseTypeTab/CareSupervision/4_DraftOrders.json:32]: yeah, true for draft, but similar situation in orders

**Review by tomaszpowroznik — APPROVED** [review]

**Pull request merged**